integration/proxy/teleterm_test.go

```diff
@@ -95,11 +95,34 @@ func testTeletermDbGatewaysCertRenewal(t *testing.T, pack *dbhelpers.DatabasePac
 		databaseURI := uri.NewClusterURI(profileName).
 			AppendDB(pack.Root.MysqlService.Name)
 
-		testDBGatewayCertRenewal(ctx, t, pack, albProxy.Addr().String(), databaseURI)
+		testDBGatewayCertRenewal(ctx, t, pack, albProxy.Addr().String(), databaseURI, nil)
+	})
+	t.Run("root cluster with per-session MFA", func(t *testing.T) {
+		requireSessionMFAAuthPref(ctx, t, pack.Root.Cluster.Process.GetAuthServer(), "127.0.0.1")
+		webauthnLogin := setupUserMFA(ctx, t, pack.Root.Cluster.Process.GetAuthServer(), pack.Root.User.GetName(), "127.0.0.1")
+
+		profileName := mustGetProfileName(t, pack.Root.Cluster.Web)
+		databaseURI := uri.NewClusterURI(profileName).
+			AppendDB(pack.Root.MysqlService.Name)
+
+		testDBGatewayCertRenewal(ctx, t, pack, "", databaseURI, webauthnLogin)
+	})
+	t.Run("leaf cluster with per-session MFA", func(t *testing.T) {
+		requireSessionMFAAuthPref(ctx, t, pack.Root.Cluster.Process.GetAuthServer(), "127.0.0.1")
+		requireSessionMFAAuthPref(ctx, t, pack.Leaf.Cluster.Process.GetAuthServer(), "127.0.0.1")
+		webauthnLogin := setupUserMFA(ctx, t, pack.Root.Cluster.Process.GetAuthServer(), pack.Root.User.GetName(), "127.0.0.1")
+
+		profileName := mustGetProfileName(t, pack.Root.Cluster.Web)
+		leafClusterName := pack.Leaf.Cluster.Secrets.SiteName
+		databaseURI := uri.NewClusterURI(profileName).
+			AppendLeafCluster(leafClusterName).
+			AppendDB(pack.Leaf.MysqlService.Name)
+
+		testDBGatewayCertRenewal(ctx, t, pack, "", databaseURI, webauthnLogin)
 	})
 }
 
-func testDBGatewayCertRenewal(ctx context.Context, t *testing.T, pack *dbhelpers.DatabasePack, albAddr string, databaseURI uri.ResourceURI) {
+func testDBGatewayCertRenewal(ctx context.Context, t *testing.T, pack *dbhelpers.DatabasePack, albAddr string, databaseURI uri.ResourceURI, webauthnLogin libclient.WebauthnLoginFunc) {
```

Comment:
It might be a good idea to define `dbGatewayCertRenewalParams`, similar to `kubeGatewayCertRenewalParams`, and use it here instead of a bunch of positional args.